db/migrate/20211115203216_add_index_on_users_peristence_token.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexOnUsersPeristenceToken < ActiveRecord::Migration[6.0]
+  def change
+    add_index :users, [:persistence_token], unique: true
+  end
+end
```

Comment:
This adds a unique index on `users.persistence_token`, as recommended by Authlogic:
https://github.com/binarylogic/authlogic#2a1-the-users-table

Authlogic already enforces unique tokens in the Rails code, but we also need the restriction in the db.  I discovered that this was missing after debugging some strange behavior in the specs, caused by users with duplicate tokens.

Reply:
Got it.  I remember you talking about this one.